package.json

```diff
@@ -32,13 +32,13 @@
   "size-limit": [
     {
       "path": "packages/components/lib/**/*([a-zA-Z-_]).js",
-      "limit": "70KB",
+      "limit": "140KB",
```

Comment:
Unrelated to this change, but why are we using a forked version of size-limit-action? 🤔 

Something to think about for the future is seeing if we can get the non-forked version working for us.

Reply:
Ah, looks like Devin added some code to not post more than 1 comment from the action: https://github.com/chanzuckerberg/size-limit-action/commit/afba6038fb5322674ca9669c5482a47aac7a31c0